File: problems/001/001.f90
PROGRAM F001

    ! Copyright 2021 Melwyn Francis Carlo

    IMPLICIT NONE

    INTEGER, PARAMETER :: VAL_1A = FLOOR((1000.0 - 1.0) / 3.0);
    INTEGER, PARAMETER :: VAL_1B = 3 * ((VAL_1A * (VAL_1A + 1)) / 2);
    INTEGER, PARAMETER :: VAL_2A = FLOOR((1000.0 - 1.0) / 5.0);
    INTEGER, PARAMETER :: VAL_2B = 5 * ((VAL_2A * (VAL_2A + 1)) / 2);
    INTEGER, PARAMETER :: VAL_3A = FLOOR((1000.0 - 1.0) / 15.0);
    INTEGER, PARAMETER :: VAL_3B = 15 * ((VAL_3A * (VAL_3A + 1)) / 2);

    PRINT *, NEW_LINE('1'), NEW_LINE('1'),  &
             VAL_1B + VAL_2B - VAL_3B,      &
             NEW_LINE('1'), NEW_LINE('1')

END PROGRAM F001

File: problems/001/001.c
#include <stdio.h>
#include <math.h>

/* Copyright 2021 Melwyn Francis Carlo */

int main()
{
    int val_1a = floor((1000-1) / 3);
    int val_1b = 3 * ((val_1a * (val_1a + 1)) / 2);
    int val_2a = floor((1000-1) / 5);
    int val_2b = 5 * ((val_2a * (val_2a + 1)) / 2);
    int val_3a = floor((1000-1) / 15);
    int val_3b = 15 * ((val_3a * (val_3a + 1)) / 2);

    printf("%d\n", val_1b + val_2b - val_3b);

    return 0;
}
